tourney/achievements/threetwo_win_achievement.py

```diff
@@ -0,0 +1,29 @@
+from .tiered_achievement import TieredAchievement
+from .behavior import WIN_BEHAVIOR
+
+class ThreeVTwoWinAchievement(TieredAchievement):
+  def __init__(self):
+    tiers = (
+      (1,   "Strength in Numbers", "In a three-player team, beat a two player team."),
+      (10,  "E Pluribus", "In a three-player team, beat a two player team ten times."),
+      (50,  "My Name is Legion", "In a three-player team, beat a two player team fifty times."),
```

Comment:
Maybe "I am Legion"?

Reply:
https://www.biblegateway.com/passage/?search=Mark%205:9&version=NIV but yeah, that form is probably more widely recognized.

Reply:
Oh yeah, I like your version better actually.